packages/design-system/src/components/Dropdown/Select/AsyncSelect/AsyncSelect.tsx

```diff
@@ -64,7 +64,11 @@ export const AsyncBaseDropdown = forwardRef<
 
     return (
       <div>
-        <Label text={label} disabled={disabled} />
+        <Label
+          text={label}
+          disabled={disabled}
+          belongsToCompleteInputField={true}
```

Comment:
You can shorten this in each place just `belongsToCompleteInputField`
```suggestion
          belongsToCompleteInputField
```